dashboard-app/src/features/dashboard/Dashboard/index.tsx

```diff
@@ -16,6 +17,30 @@ const H1 = styled(Heading1)`
 /**
  * Component
  */
+
+const testNumberProps = {
```

Comment:
I've pulled and had a look locally so this can be removed now 👍

Reply:
Can we still rebase our PRs onto one another so the feature work is contained in one commit?

Reply:
Yeah I've rebased onto yours, but haven't pushed yet